content/docs/command-reference/exp/apply.md

```diff
@@ -14,20 +14,21 @@ positional arguments:
 
 ## Description
 
-Restores the results of a given `experiment` (name or hash are accepted, see
-`dvc exp run` for details) into the workspace. This means changing the
-appropriate [metafiles](/doc/user-guide/project-structure),
-<abbr>parameter</abbr> files, <abbr>metrics</abbr>, <abbr>plots</abbr>, and
-corresponding DVC-tracked data.
+Restores a given `experiment` (name or hash are accepted, see `dvc exp run` for
```

Comment:
Could we make the statement in parentheses a new sentence?

Restores a given `experiment` into the workspace. Experiment can be referenced by a hash or name.

Reply:
Doesn't seem like a sentence pertaining to that paragraph to me. Would have to be another p just for the argument (and maybe some of the options), which I'm trying to avoid.

Reply:
I agree it would be easier to read though...